.azure-pipelines/azure-pipelines-backend.yml

```diff
@@ -3,43 +3,95 @@ trigger:
     include:
       - master
   paths:
-    exclude: 
+    exclude:
       - /.azure-pipelines/azure-pipelines-frontend.yml
       - /packages/client/*
       - /packages/client-e2e/*
 
 pool:
   vmImage: 'ubuntu-latest'
 
-steps: 
+steps:
   - task: NodeTool@0
     inputs:
       versionSpec: '14.6.0'
     displayName: 'Install Node.js 14.6.0'
 
+  # This is included in package.json, but only under devDependencies, so this (production!) build won't see it.
+  # As a workaround, install it globally
+  - script: yarn global add wasm-pack
+    displayName: 'Install wasm-pack'
+
+  - task: Cache@2
+    inputs:
+      # Cache the tarred node_modules.xz file.
+      # Note that the cache key is the hash of of yarn.lock. If the lockfile changes, the cache gets invalidated.
+      key: 'v2 | yarn | "$(Agent.OS)" | $(Build.SourcesDirectory)/yarn.lock'
```

Comment:
Had to version the key, or it would keep looking for the old node_modules cache, which was a folder containing, well, node modules. We don't want to cache that! It takes forever to publish it, and it _also_ takes forever to tar-then-publish it. So instead, the thing that we cache is a pre-tarred node_modules.